shimmy/dm_env_wrapper.py

```diff
@@ -35,21 +37,18 @@ def dm_spec2gym_spec(spec):
             raise ValueError(f"Unknown dtype {spec.dtype} for spec {spec}.")
 
         return spaces.Box(low=low, high=high, shape=spec.shape, dtype=spec.dtype)
-    elif type(spec) is dm_env.specs.DiscreteArray:
+    elif type(spec) is DiscreteArray:
         return spaces.Discrete(spec.num_values)
     else:
         raise NotImplementedError(
-            f"Unknown spec {spec} for environment, not converting."
+            f"Unknown spec {spec} that is unknown how to convert to Gymnasium space, please report."
```

Comment:
I think this sentence can be fixed abit.

Reply:
`Unknown spec {spec} with unknown conversion to Gymnasium space, please report this.`